src/streamlink/plugins/dogan.py

```diff
@@ -14,29 +15,52 @@ class Dogan(Plugin):
     Support for the live streams from Doğan Media Group channels
     """
     url_re = re.compile(r"""
-        https?://(?:www.)?
-        (?:teve2.com.tr/(?:canli-yayin|filmler/.*|programlar/.*)|
-           kanald.com.tr/.*|
-           cnnturk.com/canli-yayin|
-           dreamtv.com.tr/canli-yayin|
-           dreamturk.com.tr/canli)
+        https?://(?:www\.)?
+        (?:cnnturk\.com/(?:canli-yayin|video/.*)|
+           dreamturk\.com\.tr/(?:canli|canli-yayin-izle)|
+           kanald\.com\.tr/.*|
+           teve2\.com\.tr/(?:canli-yayin|filmler/.*|programlar/.*))
     """, re.VERBOSE)
-    playerctrl_re = re.compile(r'''<div[^>]*?ng-controller=(?P<quote>["'])(?:Live)?PlayerCtrl(?P=quote).*?>''', re.DOTALL)
+    playerctrl_re = re.compile(r'''<div\s+id="video-element".*>''', re.DOTALL)
     data_id_re = re.compile(r'''data-id=(?P<quote>["'])(?P<id>\w+)(?P=quote)''')
-    content_id_re = re.compile(r'"content(?:I|i)d", "(\w+)"')
+    content_id_re = re.compile(r'"content(?:I|i)d",\s*"(\w+)"')
```

Comment:
`(?:I|i)` -> `[Ii]`

Reply:
Ah, yes.  I must admit, I blindly copied the previous approach, but that's nicer.  Good catch on the non-greedy match for the div too.